hazelcast/src/test/java/com/hazelcast/client/impl/ClientDelegatingFutureTest.java

```diff
@@ -176,6 +191,16 @@ public void thenCombine() {
         assertEquals(DESERIALIZED_VALUE, nextStage.join());
     }
 
+    @Test
+    public void thenCombineAsync() {
```

Comment:
`CompletionStage` API methods of `ClientDelegatingFuture` are covered by `ClientDelegatingFutureTest_CompletionStageTest` which was introduced at a later point; probably it makes sense to remove all `CompletionStage` API tests from `ClientDelegatingFutureTest`?